### PR TITLE
ci(sync-files): update sync files

### PR DESCRIPTION
**Pull request by shmpwk**
## PR Type

- New Feature

## Related Links

https://github.com/tier4/autoware_launch-old/pull/800

[https://github.com/tier4/autoware_launch/commit/ac5f9b77d1fc343eaabe46e7d5b1c9bc727e0c70#diff-dccdd7eeb61ae857a571ee7[…]47f4096980fe75a6c3e30708b39e](https://github.com/tier4/autoware_launch/commit/ac5f9b77d1fc343eaabe46e7d5b1c9bc727e0c70#diff-dccdd7eeb61ae857a571ee739962d62d407147f4096980fe75a6c3e30708b39e)

## Description

Update sync files

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [x] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [x] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: .github/sync-files.yaml [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+    - source: DISCLAIMER.md
+    - source: LICENSE
+    - source: .github/ISSUE_TEMPLATE/bug.yaml
+    - source: .github/ISSUE_TEMPLATE/config.yml
+    - source: .github/ISSUE_TEMPLATE/task.yaml
+    - source: .github/PULL_REQUEST_TEMPLATE.md
+    - source: .github/PULL_REQUEST_TEMPLATE/small-change.md
+    - source: .github/PULL_REQUEST_TEMPLATE/standard-change.md
     - source: .github/dependabot.yaml
-    - source: .github/workflows/backport.yaml
```
Comment (shmpwk) [inline .github/sync-files.yaml:4]: Need to check .github/workflows/backport.yaml

Comment (shmpwk) [reply .github/sync-files.yaml:4]: Need backport
https://github.com/tier4/autoware_launch-old/pull/477

Comment (shmpwk) [reply .github/sync-files.yaml:4]: Add it by https://github.com/tier4/autoware_launch/pull/28/commits/a6d8f1bd050cce4525eff49d661b00ed7773cd7a

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**